device/rsvp/rsvpInterface.yaml

```diff
@@ -0,0 +1,81 @@
+components:
+  schemas:
+    Rsvp.Interface:
+      x-status: under-review
+      description: >-
+        Configuration for RSVP Interface.
+      type: object
+      required: [ipv4_name,neighbor_ip]
+      properties:
+        ipv4_name: 
+          description: >-
+            The globally unique name of the IPv4 interface connected to the DUT.            
+          type: string
+          x-constraint:
+          - '/components/schemas/Device.Ipv4/properties/name'
+        neighbor_ip:
+          description: >-
+            IPv4 address of the RSVP neighbor on this interface.
+          type: string
+          format: ipv4
+        label_space_start:
+          description: >-
+            The user-defined label space start value. The LSPs for which this router acts as a egress are assigned labels from this
+            label pool. 
+          type: integer
+          default: 1000
+          minimum: 0
+          maximum: 1048575
+        label_space_end:
+          description: >-
+            The user-defined label space end value.
+          type: integer
+          default: 100000
+          minimum: 0
+          maximum: 1048575
+        enable_refresh_reduction:
+          description: >-
+            Enables sending of Refresh Reduction as described in RFC2961.
+          type: boolean
+          default: false
+        summary_refresh_interval:
+          description: >-
+            The number of seconds between transmissions of successive Summary Refreshes.
+            There is no specification specified maximum value. For clarity, setting the maximum to 1 hour.
+          type: integer          
```

Comment:
unsigned integer

Reply:
There are many integers for which negative value does not make sense. I am keeping only one place-holder comment here for all the numeric values that are not floating points.

Reply:
unsigned ints are represented as int64 in the model